server/src/main/java/org/elasticsearch/action/admin/indices/rollover/MetadataRolloverService.java

```diff
@@ -88,16 +88,16 @@ private RolloverResult(String rolloverIndexName, String sourceIndexName, Cluster
 
     public RolloverResult rolloverClusterState(ClusterState currentState, String rolloverTarget, String newIndexName,
                                                CreateIndexRequest createIndexRequest, List<Condition<?>> metConditions,
-                                               boolean silent) throws Exception {
+                                               boolean silent, boolean onlyValidate) throws Exception {
```

Comment:
This is currently only invoked with `true, true` or `false, false`. Is the reason for this that you want to reuse it later as `true, false` for autoscaling?

Reply:
Yes, that is the intention though there is follow-up work to ensure that the allocation-service logging does not occur for autoscaling.